ci/check_enterprise_merge.sh

```diff
@@ -12,7 +12,7 @@ set -e
 set -o pipefail
 
 PR_BRANCH="${CIRCLE_BRANCH}"
-ENTERPRISE_REMOTE="https://${GIT_USERNAME}:${GIT_TOKEN}@github.com/citusdata/citus-enterprise"
```

Comment:
It might make sense to remove this check from CI in a separate PR as we do not have an enterprise version for recent versions of Citus.